commisery/rules.py

```diff
@@ -287,12 +257,15 @@ def C012_missing_type_tag(
         )
 
 
-def C013_subject_should_not_end_with_punctuation(
-    message: CommitMessage, _: Configuration
-):  # pylint: disable=C0103
+def C013_subject_should_not_end_with_punctuation(message: COMMIT_TYPE, config: Configuration):
     """The commit message's subject should not end with punctuation"""
     # No need to verify merge commits
-    if message.is_merge():
+    if _is_acceptable_merge_message(message):
+        return
+
+    try:
+        C009_missing_description(message, config)
```

Comment:
This line surprises me a bit, why would the rule that states that subject shouldn't end with punctuation do something with description? Shouldn't this call be part of some control logic that chains the rules?

Reply:
A similar comment was shared on a prior review - I have a task open to rework this 'dependency'-setup for all of the rules